include/openenclave/edger8r/common.h

```diff
@@ -133,12 +133,12 @@ OE_INLINE oe_result_t oe_add_size(size_t* total, size_t size)
 /**
  * Copy an input parameter to input buffer.
  */
-#define OE_WRITE_IN_PARAM(argname, argsize, argtype)                     \
-    if (argname)                                                         \
-    {                                                                    \
-        _args.argname = (argtype)(_input_buffer + _input_buffer_offset); \
-        OE_ADD_SIZE(_input_buffer_offset, (size_t)(argsize));            \
-        memcpy((void*)_args.argname, argname, (size_t)(argsize));        \
+#define OE_WRITE_IN_PARAM(argname, argsname, argsize, argtype)            \
```

Comment:
Oops, I can probably completely undo this change. I don't believe it's necessary any more.